src/js/chrome.js

```diff
@@ -8,7 +9,7 @@ const libjwt = auth();
 function noop () {}
 
 libjwt.initPromise.then(() => {
-    libjwt.jwt.getUserInfo().then(analytics).catch(noop);
+    libjwt.jwt.getUserInfo().then(analytics && sentry).catch(noop);
```

Comment:
I don't think that this is going to work. It's going to run analytics and then dies. To fix this you should do something like
```JS
libjwt.jwt.getUserInfo().then((...data) => {
  analytics(...data);
  sentry(...data);
}).catch(noop);
```
This way any data returned ffrom getUser info will be passed in same way.

Reply:
Did you test the code locally? At least for me, it was hitting both analytics and sentry using this method.

Reply:
Oh `analytics` returns true? In that case we are good with shortcutting.